src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/WorkingDayCountService.kt

```diff
@@ -1,37 +1,33 @@
 package uk.gov.justice.digital.hmpps.approvedpremisesapi.service
 
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.core.io.Resource
 import org.springframework.stereotype.Service
-import uk.gov.justice.digital.hmpps.approvedpremisesapi.client.ClientResult
-import uk.gov.justice.digital.hmpps.approvedpremisesapi.client.GovUKBankHolidaysApiClient
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.bankholidaysapi.UKBankHolidays
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.getDaysUntilInclusive
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.getNextWorkingDay
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.isWorkingDay
 import java.time.LocalDate
 
 @Service
 class WorkingDayCountService(
-  private val govUKBankHolidaysApiClient: GovUKBankHolidaysApiClient,
+  @Value("classpath:data/bank-holidays.json")
+  private val bankHolidayJson: Resource,
+  private val objectMapper: ObjectMapper,
 ) {
-
   fun getWorkingDaysCount(from: LocalDate, to: LocalDate): Int {
-    val bankHolidays = when (val govUKBankHolidaysResponse = govUKBankHolidaysApiClient.getUKBankHolidays()) {
-      is ClientResult.Success -> govUKBankHolidaysResponse.body.englandAndWales.events.map { it.date }
-      is ClientResult.Failure -> govUKBankHolidaysResponse.throwException()
-    }
-    return from.getDaysUntilInclusive(to).filter { it.isWorkingDay(bankHolidays) }.size
+    return from.getDaysUntilInclusive(to).filter { it.isWorkingDay(this.getBankHolidays()) }.size
```

Comment:
So this says "Get the days from `from` until `to`. but filter out any BHs and weekends"?